### ISSUE TITLE
Feeding the generated "filter images" from convnet2D to individuals LSTMs (?) is it possible ?

### ISSUE THREAD

**Issue by rjpg** [keras-team/keras#8878]
hello I wanted to feed several LSTMs (that has input 2D) with the output of a convnet2D (that has output dimension 3D [#filter image, width, hight]).

**Comment by rjpg** [comment]
Maybe the solution comes from here :
https://github.com/keras-team/keras/issues/890

to split/slice the tensor from [2,5,5] to [5,5] [5,5]  (?)

But I am not sure how to apply it ....

**Comment by gabrieldemarmiesse** [comment]
Hi! You can! Just use a Reshape layer to make the tensor be 2D instead of 3D.

Don't forget to close this issue and remember that stackoverflow is more appropriate for questions like this. 
Keras github issues are to report bugs in the keras codebase. Thank you!

**Comment by rjpg** [comment]
Hello thanks for the reply (rechape only does not work). however I have solve the problem but the results are not that good ... here it is : 

https://github.com/rjpg/bftensor/blob/master/Autoencoder/src/RJPGNet.ipynb

I guess "ensemble on-the-fly" is not that good I will do-it step by step ... conv (train with autoencoder ) then each "filtered image" (from the encoder/middle layer) to one lsts and then essemble ... 3 separate trains

**Comment by gabrieldemarmiesse** [comment]
If you want your lstm to be able to communicate with each others, you need to use only one lstm layer. Bonus: it'll be faster.

out1=Dropout(0.40)(conv2)
out1 = Reshape(5,25)(out1)
out_lstm  = LSTM(40,input_shape=(5,25))(out1)
this way the lstms are processing image after image. Unlike your code where each image is processed independantly (thus making the lstm layer no more useful than a dense layer).

**Comment by rjpg** [comment]
I was thing of putting all images, that represent the time series (time sample x variables ) in one like 5x5x5  (number of "filtered images" x time samples (compressed by conv layers) x variables  )into a (25 x 5) . Thys way flatten the variables and keet the time sequence. but this is not a linear reshape ... Iam seeing how to do that ...

your idea has a flaw : that way  I will be feeding 5 sequences into a LSTM each iteration (not good for lstm). The goal, if we want to have one LSTM, is to feed one sequece of 5 time-samples and 25 variables (5x5 : 5 from each "filtered image" line from the conv layers)

Do you understand ? 

the reshape has to be something like this (ex 3x3x3 into 9x3):
filter img1  filter img2  filter img3
[[1,2,3],       [[4,5,6],     [[7,8,9]
[1,2,3],        [4,5,6]        [7,8,9]    
[1,2,3]]        [4,5,6]]      [7,8,9]]

intended result :
[[1,2,3,4,5,6,7,8,9]
[1,2,3,4,5,6,7,8,9]
[1,2,3,4,5,6,7,8,9]]

**Comment by gabrieldemarmiesse** [comment]
I'm sorry then, I don't really know how to do that.
I hope you find the answer!

**Comment by rjpg** [comment]
hello, 

I came across this that refers to the swapaoxes to join the images with out mixin dimensions ...


https://stackoverflow.com/questions/36905288/how-would-you-reshape-a-collection-of-images-from-numpy-arrays-into-one-big-imag

BUT I am not seeing how to use it to put the output of the convnet "filtered images" side by side ?

If you can help me using this swapaoxes  to do the appropriate reshape it would be nice !
